java-app-jcache/src/main/liberty/config/server.xml

```diff
@@ -2,12 +2,12 @@
 <server description="defaultServer">
     <!-- Enable features -->
     <featureManager>
-        <feature>cdi-2.0</feature>
-        <feature>jaxb-2.2</feature>
-        <feature>jsf-2.3</feature>
-        <feature>jaxrs-2.1</feature>
-        <feature>ejbLite-3.2</feature>
+        <feature>cdi-4.0</feature>
+        <feature>enterpriseBeansLite-4.0</feature>
```

Comment:
Seems missed `<feature>jsonb-3.0</feature>`, see https://github.com/Azure-Samples/open-liberty-on-aks/pull/44/files#diff-5544fdf9615f35e8bddf8bad71e3242c09b034438c10097322ea3ff755159a2eR6

Reply:
Fixed in [1671fd1](https://github.com/Azure-Samples/open-liberty-on-aks/pull/47/commits/1671fd1b288d88918bdabae4478b7a39458021cb).